network/network.go

```diff
@@ -130,7 +130,10 @@ func createConsensusGenesis(pv *privval.FilePV) ([]byte, error) {
 		PubKey:  pv.GetPubKey(),
 		Power:   10,
 	}}
-	
+
+	// Using less than 1 second TimeIota to simulate a consensus invalid block time 
+	genDoc.ConsensusParams.Block.TimeIotaMs = 950
```

Comment:
[question] Don't you think standalone should be configured in the same way Sirius/Mainnet is by default? I see you changed all the values to make this particular issue fail to validate the solution but I don't think it should be part of the master and affecting rest of the behaviour. WDYT?

Reply:
I agree but having a different consensus param does not make Standalone different than Sirius or Mainnet, it runs exactly the same network BUT in a more extreme and demanding framework. I do not feel very strong about that change, therefore if you do, I am happy to rever that.

Regarding the tracing, it is not very possible to do that without implementing a large refactor in the way Standalone is initialized and also in what methods and data are exposed. If you consider this a priority, we could discuss how to do it and write some tracing code around this logic.

Reply:
I was reviewing more the tracing and it would indeed require a refactor including changes on Tendermint side... alright, let's leave Standalone with these high performance settings. At the end each change must be tested on Sirius as well so in case something would behave differently on Standalone we would catch it on Sirius 👍